iroh/src/downloader/get.rs

```diff
@@ -223,9 +223,6 @@ pub async fn get<D: Store>(
         BlobFormat::Raw => get_blob(db, conn, &hash).await,
         BlobFormat::HashSeq => get_hash_seq(db, conn, &hash).await,
     };
-    if let Err(e) = stats.as_ref() {
-        tracing::error!("get failed: {e:?}");
-    }
```

Comment:
why is this removed?

Reply:
because it would log an error for each failed download attempt, while there might still be other peers queued, and I thought an `error!` log for each failed attempt is too much. if the final attempt fails, a `warn!` is logged already in `downloader.rs`.